apps/react-18-tests-v9/src/App.cy.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { mount } from '@cypress/react';
+
+import { App } from './App';
+
+const providerSelector = '.fui-FluentProvider';
+
+describe('App with React 18', () => {
```

Comment:
propagated these test to Root as they are rendering the App instead of manually re-creating Provider